code/modules/mob/living/life.dm

```diff
@@ -66,6 +69,13 @@
 /mob/living/proc/handle_chemicals_in_body()
 	return
 
+/mob/living/proc/handle_diginvis()
+	var/image/hide = image(loc = src)
+	hide.override = 1
+	for(var/mob/living/silicon/ai/AI in player_list)
+		AI.client.images += hide
```

Comment:
NO, GOD NO.

this is a looping proc that repeatedly makes /images, and targets all AIs (this includes ones targetted by a previous iteration of the loop) so ```+=``` is unacceptable.

give the mob a var to keep track of it's hidden image, and ```|=``` the image to AIs, instead of ```+=```
Alternatively, reuse the static overlay perhaps?